data/human/deep missions.txt

```diff
@@ -3320,15 +3316,15 @@ mission "Home for Skadenga 4"
 				`	"That's fantastic news! The Skakenga will need transport to Windblain."`
 				`	"They need to go to Windblain."`
 			`	A hush falls over the room, and both Braiden and Grayden look to Kaiden with confused expressions on their faces.`
-			`	"Wind. Blain." Kaiden says slowly. "Wiiiindblaaain. Are you sure that is the world these simple people wish to travel to? Maybe you got an invoice mixed up? Perhaps wrote the wrong name down?" Braiden and Grayden both silently nod.`
```

Comment:
He does not say 'Windblaine'. He says Wiiiindblaaaain. 
He's exaggerating the enunciation.